charts/pega/README.md

```diff
@@ -756,7 +756,43 @@ pegasearch:
     - name: TZ
       value: "EST5EDT"
 ```
+## Deploying Pega with external kafka for stream
 
+
+Starting from 8.7 embedded stream (Stream tier or “Stream” `nodetype`) is deprecated. Pega recommends deployment of stream with external kafka.
+Configure external Kafka as a stream service provider to use your own managed Kafka infrastructure.
+
+### Stream (External Kafka) settings
+
+```yaml
+# Stream (External Kafka) settings.
+# Disabled by default, you must not use stream tier when this setting is enabled.
```

Comment:
Can we enable by default with a working deployment config using open source Kafka charts as a dependency?

Reply:
Enabling it by default may cause data loss. I would prefer to add a reference to open source Kafka charts instead of adding them as a dependency.